packages/graph/hash_graph/lib/graph/src/store/postgres/entity/read.rs

```diff
@@ -36,3 +39,36 @@ impl<C: AsClient> crud::Read<'_, EntityId, Entity> for PostgresStore<C> {
             .change_context(QueryError)?)
     }
 }
+
+#[async_trait]
+impl<C: AsClient> crud::Read<'_, AllLatest, Entity> for PostgresStore<C> {
+    type Output = Vec<Entity>;
+
+    async fn get(&self, _: AllLatest) -> Result<Self::Output, QueryError> {
+        let row_stream = self
+            .as_client()
+            .query_raw(
+                r#"
+                    SELECT DISTINCT ON(entity_id) properties
+                    FROM entities
+                    ORDER BY entity_id, version DESC;
+                    "#,
```

Comment:
Can we change it to one of these?
```suggestion
                r#"
                SELECT DISTINCT ON(entity_id) properties
                FROM entities
                ORDER BY entity_id, version DESC;
                "#,
```
or
```suggestion
                r#"
                    SELECT DISTINCT ON(entity_id) properties
                    FROM entities
                    ORDER BY entity_id, version DESC;
                "#,
```